docs/data-collection/pre-session.md

```diff
@@ -228,27 +228,32 @@ Instructions of operations to be performed before the participant arrival, **bef
     |:--:|
     | ![start-session-syncbox](../assets/images/start-session-syncbox.png) |
     | ![run-session-syncbox](../assets/images/trigger_box.jpg) |
-    
+
+## Eye Tracker and projector setup
+
+Please note that some of the images used in this section are sourced from the [Eye Tracker Instruction Manual](https://www.sr-research.com/support/attachment.php?aid=1376)
+
+!!! warning "To ensure accurate performance and prevent minor drifts in the pupil and CR thresholds, perform the ET Setup 10-15 minutes after powering the ET on."
```

Comment:
The reader does not know at this point what is the CR threshold. Please introduce the abbreviation and explain (maybe earlier if there is somewhere else where this info would fit better in).